src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesNotShown.jsx

```diff
@@ -1,25 +1,14 @@
-import React, { useState, useEffect } from 'react';
-import PropTypes from 'prop-types';
-
-// eslint-disable-next-line import/no-unresolved
 import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
-import { GA_PREFIX } from '../../../utils/constants';
+import React, { useEffect, useState } from 'react';
 import NewTabAnchor from '../../../components/NewTabAnchor';
+import { GA_PREFIX } from '../../../utils/constants';
 
-export default function FacilitiesNotShown({ sortMethod }) {
+export default function FacilitiesNotShown() {
   const [isOpen, setIsOpen] = useState(false);
-  useEffect(
-    () => {
-      setIsOpen(false);
-    },
-    [sortMethod],
-  );
   useEffect(
     () => {
       if (isOpen) {
-        recordEvent({
-          event: `${GA_PREFIX}-facilities-not-listed-click`,
-        });
+        recordEvent({ event: `${GA_PREFIX}-facilities-not-listed-click` });
```

Comment:
FYI events are tricky to test locally so I just added a console.log here to verified when this if statement triggers.